src/ILCompiler.Compiler/src/ILCompiler.Compiler.csproj

```diff
@@ -38,6 +38,12 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="..\..\Common\src\Internal\Runtime\GCDescEncoder.cs">
+      <Link>Common\GCDescBuilder.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Common\src\Internal\Runtime\ITargetBinaryWriter.cs">
+      <Link>Common\IBinaryWriter.cs</Link>
```

Comment:
Good catch! Also fixed the GCDescBuilder just above. I did a lot of renaming trying to find something fitting. I'm still not completely happy.